### DISCUSSION TITLE
Project 1 Part 3

### DISCUSSION THREAD

**Discussion by feliciones** [cooperrc/computational-mechanics#33] [Q&A]
For the third part of this project, do we need to solve a linear ODE in order to find the numerical solution? I've been following this site as a guide for the ODE but there's gotta be an easier way, right?

[https://autarkaw.wordpress.com/2008/07/21/time-of-death-a-classic-ode-problem/]

**Comment by cooperrc** [comment]
You already have the linear ODE solution, given as the "analytical solution".

You can set the temperature, $T(t) = 98.6^oF$ and solve for $t$ in the equation
